lib/ballot_test.go

```diff
@@ -36,3 +40,28 @@ func TestErrorBallotHasOverMaxTransactionsInBallot(t *testing.T) {
 	err := ballot.IsWellFormed(networkID)
 	require.Error(t, err, sebakerror.ErrorBallotHasOverMaxTransactionsInBallot)
 }
+
+/*
+	TestBallotHash checks that ballot.GetHash() makes non-empty hash.
+*/
+func TestBallotHash(t *testing.T) {
```

Comment:
Usually in Golang, the comment of function or method was started with `//` and functions name. This comment can be changed like this,

```
// 	TestBallotHash checks that ballot.GetHash() makes non-empty hash.
```